storage/client/client.go

```diff
@@ -131,14 +135,35 @@ func NewStorageClient(chainName common.ChainName, db storage.TargetStorage, l *l
 		l.Error("api client: failed to create tx cache: %w", err)
 		return nil, err
 	}
-	return &StorageClient{chainName, db, blockCache, txCache, l}, nil
+	return &StorageClient{chainName, db, runtimeClients, networkConfig, blockCache, txCache, l}, nil
 }
 
 // Shutdown closes the backing TargetStorage.
 func (c *StorageClient) Shutdown() {
 	c.db.Close()
 }
 
+// Returns the native token symbol of the specified runtime in the network
+// specified by the networkConfig.
+func (c *StorageClient) nativeTokenSymbol(runtime common.Runtime) string {
+	if c.networkConfig == nil {
+		c.logger.Warn("no network config available; unable to determine native token symbol", "runtime", runtime)
+		return ""
+	}
+	return c.networkConfig.ParaTimes.All[string(runtime)].Denominations[oasisConfig.NativeDenominationKey].Symbol
```

Comment:
Good point; added check for missing paratime. `Denomination` is a bit less clear; all paratimes are currently configured to have a native denomination but I'm not sure whether this is required for future paratimes. Will add the check just in case.